spec/acceptance/food_truck_api_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rack/test'
+require_relative '../../app/api'
+
+module FoodTruck
+  RSpec.describe 'Food Truck API' do
+    include Rack::Test::Methods
+
+    def app
+      API.new
+    end
+
+    describe 'provides message' do
+      it 'of listed food trucks' do
```

Comment:
"it of listed food trucks" reads like broken English.  I'd recommend something like this:

``` ruby
describe 'get /api' do
  it "returns a slack-formatted list of available food trucks" do
    # ...
  end
end
```

"it returns a slack-formatted list of available food trucks" reads much more like understandable English :).